features/upgrade/monitoring/monitoring-upgrade.feature

```diff
@@ -23,7 +23,10 @@ Feature: cluster monitoring related upgrade check
   Scenario: upgrade cluster monitoring along with OCP
     Given I switch to cluster admin pseudo user
     And I use the "openshift-monitoring" project
+    And I wait up to 120 seconds for the steps to pass:
+    """
     Given all pods in the project are ready
+    """
```

Comment:
```
# use this with care and it just all of the pods in the project, better to filter out the pods with labels if possible
Given /^all pods in the project are ready$/ do
  pods = project.pods(by:user)
  ready_timeout = 15 * 60
  logger.info("Number of pods: #{pods.count}")
  pods.each do |pod|
    cache_pods(pod)
    @result = pod.wait_till_ready(user, ready_timeout)
    unless @result[:success]
      raise "pod #{pod.name} did not become ready within allowed time"
    end
  end
end
```
The step itself contains wait logic (for 15*60 seconds), I'm afraid adding another wrap for 120 seconds won't help there.

Reply:
It does help here, I need get pods again from the project for exporter pod is just deleted and I should get the deleted information, that is, wait for until the exporter pod is gone.

Reply:
Where is the deletion step of export pod ?

Reply:
the delete and adding of exporter pod is automatically triggered by node number change, when there is mcp config change, node change will happen. One node had one exporter pod.
mcp config change belong to other team's test case

Reply:
Thanks for the explanation.